pkg/metrics/tool/common.go

```
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2022. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package tool
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/containerd/containerd/log"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func FormatFloat64(f float64, point int) float64 {
+	var value float64
+	switch point {
+	case 6:
+		value, _ = strconv.ParseFloat(fmt.Sprintf("%.6f", f), 64)
+	case 2:
+		fallthrough
+	default:
+		value, _ = strconv.ParseFloat(fmt.Sprintf("%.2f", f), 64)
+	}
+
+	return value
+}
+
+func ParseFloat64(val string) float64 {
+	floatVal, err := strconv.ParseFloat(val, 64)
+	if err != nil {
+		log.L.Warnf("parse float failed, error: %v", err)
+	}
```

We should ignore error

---

Fixed.